tests/appsec/iast/sink/test_command_injection.py

```diff
@@ -49,6 +49,24 @@ def test_insecure(self):
     def setup_secure(self):
         self.sink_fixture.setup_secure()
 
-    @missing_feature(reason="Endpoint not implemented")
+    @missing_feature(library="nodejs", reason="Endpoint not implemented")
     def test_secure(self):
         self.sink_fixture.test_secure()
+
+    def setup_telemetry_metric_instrumented_sink(self):
+        self.sink_fixture.setup_telemetry_metric_instrumented_sink()
+
+    @released(dotnet="?", golang="?", java="1.13.0", nodejs="?", php_appsec="?", python="?", ruby="?")
```

Comment:
`released` decorator can only go on classes. For method, you can use directly `missing_feature` (same comment for other occurences)

Reply:
@cbeauchesne Ok. I'll change it. However, I have used this pattern before at https://github.com/DataDog/system-tests/pull/1173/files and it seemed to work?

Reply:
it works, but it's an issue :) 

Basically, the key idea is we export the information feature/released version in different dashboard spreadsheet. And it only work at feature level, which is represented by classes in system tests.

Not a big deal though, it will just save me some time when I'll fix this small API inconsistency.